src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -29,6 +29,9 @@
     <TargetingPackReference Include="System" />
     <TargetingPackReference Include="System.Core" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <TargetingPackReference Include="System.Private.CoreLib" />
```

Comment:
I think this is ok, but you all are more familiar with type forward than me ;-)
Are you saying that System.ValueTuple can/should not be direct type forward to the original (in System.Private.Corelib)?